backend/routes/server.js

```diff
@@ -108,39 +109,41 @@ app.post("/auth/signup", async (req, res) => {
 
 //log in
 app.post("/auth/signin", async (req, res) => {
-    const { email, password } = req.body;
+  const { email, password } = req.body;
 
-    fs.readFile(userDB, 'utf-8', (err, data) => {
-        if (err) {
-            console.error(err);
-            return res.status(500).json({ message: 'Error reading user data' });
-        }
-        const users = JSON.parse(data);
+  fs.readFile(userDB, 'utf-8', (err, data) => {
+    if (err) {
+      console.error(err);
+      return res.status(500).json({ message: 'Error reading user data' });
+    }
+    const users = JSON.parse(data);
 
-        const user = users.find((u) => u.userEmail === email);
-        if (!user) {
-            return res.status(401).json({ message: 'Email does not exist' });
-        }
+    const user = users.find((u) => u.userEmail === email);
+    if (!user) {
+      return res.status(401).json({ message: 'Email does not exist' });
+    }
 
-        bcrypt.compare(password, user.userPassword, (err, result) => {
-            if (err) {
-                console.error(err);
-                return res.status(500).json({ message: 'Error comparing passwords' });
-            }
-            if (!result) {
-                return res.status(401).json({ message: 'Wrong password' });
-            }
+    bcrypt.compare(password, user.userPassword, (err, result) => {
+      if (err) {
+        console.error(err);
+        return res.status(500).json({ message: 'Error comparing passwords' });
+      }
+      if (!result) {
+        return res.status(401).json({ message: 'Wrong password' });
+      }
 
```

Comment:
These changes are shown because I formatted this file, you should use prettier to do it